database/redis/README.md

```diff
@@ -0,0 +1,101 @@
+## Redis
+
+Redis is an open source (BSD licensed), in-memory data structure store, used as a database, cache and message broker. It supports data structures such as strings, hashes, lists, sets, sorted sets with range queries, bitmaps, hyperloglogs, geospatial indexes with radius queries and streams.
+
+In this example, we will generate a Singularity container running Redis server v5.0.0 over Debian 9 (Stretch).
+
+To run this example you will need:
+
+ - Singularity, which you can download and install from [here](https://github.com/sylabs/singularity).
+ - A text editor, like: `micro`, `vim`, or `nano`.
+ - Root access.
+
+We'll go through the explanation of every section in the definition file:
+
+In the `%help` section, you can find the description for the container with the respective version of Redis installed (5.0.0) for the specific OS distribution.
+
+In the `%post` section, all the needed dependencies are installed to make Redis work properly.
+
+In the `%startscript` section, the commands needed to run an instance of the Redis server are called. We will use this section to start the Redis server daemon by default in `127.0.0.1:6379`.
+
+In the `%runscript` section, the commands needed to run a container are called. In this case we will use this section to call the Redis Client and interact with the database.
+
+#### Build your Redis container using Singularity:
+
+We will work in a directory called ` ~/redis`, if you don't have it create it and inside get the definition file:
+
+```
+$ mkdir  ~/redis
+$ cd  ~/redis
+```
+
+To obtain the definition file just run:
+
+```
+$ wget https://raw.githubusercontent.com/sylabs/examples/master/database/redis/redis.def
+```
+
+After that you could simply run the following command:
+
+```
+$ sudo singularity build redis.sif redis.def
+```
+
+You can also pull the container from our library like so:
+
+```
+$ singularity pull library://sylabs/examples/redis:latest
```

Comment:
Add that you can use the `verify` command to verify that the container comes from us and hasn't been messed with.  You can ask @WestleyK to sign the container.  And then include a link to our docs about signing and verifying your own containers.  Look at the the octave PR for an example.